pkg/status/cel-evaluator.go

```diff
@@ -48,7 +54,8 @@ func newCELEvaluator() (*celEvaluator, error) {
 		cel.Declarations(
 			decls.NewVar(sourceObjectKey, decls.NewMapType(decls.String, decls.Dyn)),
 			decls.NewVar(returnedKey, decls.NewMapType(decls.String, decls.Dyn)),
-			decls.NewVar(inventoryKey, decls.NewMapType(decls.String, decls.String)),
+			decls.NewVar(inventoryKey, decls.NewMapType(decls.String, decls.String)), // contains name:string only
```

Comment:
Not new in this PR, but FYI: it is possible to also declare that this map has only one entry (with key = "name"), and that should be done (not necessarily in this PR).